request_test.go

```diff
@@ -50,7 +50,11 @@ func testVersionDecodable(t *testing.T, name string, out versionedDecoder, in []
 }
 
 func testRequest(t *testing.T, name string, rb protocolBody, expected []byte) {
```

Comment:
I don't follow? All I did was split `testRequest` into the two pieces `testRequestEncode` and `testRequestDecode`; this method just calls them both.